src/data-loaders/RelationalDataLoader.js

```diff
@@ -94,12 +95,38 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
       label: row.name,
       name: row.name,
       type: row.dataTypeID,
-      typeName,
+      typeName: typeName,
```

Comment:
thank you for making the changes. I've approved the PR. If possible, please replace `typeName: typeName` with just `typeName`